machine/delta/delta_wb2448/kernel/acpi_enforce.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/drivers/acpi/osl.c b/drivers/acpi/osl.c
+index de03938..fe12708 100644
+--- a/drivers/acpi/osl.c
++++ b/drivers/acpi/osl.c
+@@ -1610,7 +1610,7 @@ __setup("acpi_no_auto_serialize", acpi_no_auto_serialize_setup);
+ #define ENFORCE_RESOURCES_LAX    1
+ #define ENFORCE_RESOURCES_NO     0
+ 
+-static unsigned int acpi_enforce_resources = ENFORCE_RESOURCES_STRICT;
++static unsigned int acpi_enforce_resources = ENFORCE_RESOURCES_NO;
```

Comment:
Is that perhaps then a bug with your ACPI tables for this box?  I would not make this change without understanding what that error message is telling me.  Why is there a conflict?